src/api/operator/utils.ts

```diff
@@ -0,0 +1,54 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+// TODO: move to const or app settings?
+const FILLED_ORDER_EPSILON = '0.0001' // == 0.01%
+
+function isOrderFilled(order: RawOrder): boolean {
+  let amount, executedAmount
+
+  if (order.kind === 'buy') {
+    amount = new BigNumber(order.buyAmount)
+    executedAmount = new BigNumber(order.executedBuyAmount)
+  } else {
+    amount = new BigNumber(order.sellAmount)
+    executedAmount = new BigNumber(order.executedSellAmount)
+  }
+
+  const minimumAmount = amount.multipliedBy(ONE_BIG_NUMBER.minus(FILLED_ORDER_EPSILON))
+
+  return executedAmount.gte(minimumAmount)
+}
+
+function isOrderExpired(order: RawOrder): boolean {
+  return Math.floor(Date.now() / 1000) > order.validTo
+}
+
+function isOrderPartiallyFilled(order: RawOrder): boolean {
+  if (order.kind === 'buy') {
```

Comment:
i won't mention these again in this review but I think we should use an `OrderType` `enum`